src/main/shell-session.ts

```diff
@@ -110,6 +110,14 @@ export class ShellSession extends EventEmitter {
       env["SystemRoot"] = process.env.SystemRoot;
       env["PTYSHELL"] = process.env.SHELL || "powershell.exe";
       env["PATH"] = pathStr;
+      env["LENS_SESSION"] = "true";
+      const lensWslEnv = "KUBECONFIG/up:LENS_SESSION/u";
+
+      if (process.env.WSLENV != undefined) {
+        env["WSLENV"] = `${process.env["WSLENV"]}:${lensWslEnv}`;
+      } else {
+        env["WSLENV"] = lensWslEnv;
+      }
```

Comment:
```suggestion
      env["WSLENV"] = [
        process.env.WSLENV,
        "KUBECONFIG/up:LENS_SESSION/u"
      ].filter(Boolean).join(":");
```

Reply:
Initially I made something like this but ended up to have the current implementation for the sake of more readable code.

Reply:
okay fair, I find this more readable because there no `if` that has to be deciphered for the differences and each value is treated the same.